R/map2.R

```diff
@@ -64,11 +64,16 @@ map2_chr <- function(.x, .y, .f, ..., .progress = NULL) {
   .Call(map2_impl, environment(), ".x", ".y", ".f", "character", .progress)
 }
 
+#' @rdname map2
+#' @export
+map2_vec <- function(.x, .y, .f, ..., .ptype = NULL, .progress = NULL) {
```

Comment:
Why is `.progress`'s default `NULL` instead of `FALSE` as in the other functions?

Reply:
It seems to happen for all map2 variants. I added that to https://github.com/tidyverse/purrr/issues/947